src/content/self-hosted/administration/configuration.mdx

```diff
@@ -1010,6 +1010,33 @@ wildcardCertificate:
       key: "ca.crt"
 ```
 
+### user
+
+#### serviceAccount
+
+The configured annotations and labels will be applied to each service account created by Okteto.
+These annotations and labels are additional to the ones already applied by Okteto.
+
+- `annotations`: Annotations applied to each service account generated by Okteto for user accounts.
```

Comment:
I think this description is redundant with the first paragraph. maybe we remove the first sentence ("The configured annotations and labels will be applied to each service account created by Okteto." and we just leave the explanation for annotations and labels?

Reply:
I've reworded this part, let me know what you think